readthedocs/proxito/tests/responses/v0.json

```diff
@@ -0,0 +1,120 @@
+                "name": "project",
+                "programming_language": {
+                    "code": "words",
+                    "name": "Only Words"
+                },
+                "homepage": "http://project.com",
+                "repository": {
+                    "type": "git",
+                    "url": "https://github.com/readthedocs/project"
+                },
+                "slug": "project",
+                "subproject_of": null,
+                "tags": [
+                    "project",
+                    "tag",
+                    "test"
+                ],
+                "translation_of": null,
+                "users": [
+                    {
+                    "username": "testuser"
+                }
+                ]
+            },
+            "version": {
+                "active": true,
+                "hidden": false,
+                "built": true,
+                "downloads": {},
+                "id": 1,
+                "identifier": "a1b2c3",
+                "ref": null,
+                "slug": "latest",
+                "type": "tag",
+                "verbose_name": "latest"
+            },
+            "build": {
+                "commit": "a1b2c3",
+                "created": "2019-04-29T10:00:00Z",
+                "duration": 60,
+                "error": "",
+                "finished": "2019-04-29T10:01:00Z",
+                "id": 1,
+                "project": "project",
+                "state": {
+                    "code": "finished",
+                    "name": "Finished"
+                },
+                "success": true,
+                "version": "latest"
+            },
+            "domains": {
+                "dashboard": "readthedocs.org"
+            },
+            "readthedocs": {
+                "analytics": {
+                    "code": null
+                }
+            },
+            "features": {
+                "analytics": {
+                    "code": null
+                },
+                "external_version_warning": {
+                    "enabled": true,
+                    "query_selector": "[role=main]"
+                },
+                "non_latest_version_warning": {
+                    "enabled": true,
+                    "query_selector": "[role=main]",
+                    "versions": [
+                        "latest"
+                    ]
+                },
+                "doc_diff": {
+                    "enabled": true,
+                    "base_url": "https://project.dev.readthedocs.io/en/latest/index.html",
+                    "root_selector": "[role=main]",
+                    "inject_styles": true,
+                    "base_host": "",
+                    "base_page": ""
+                },
+                "flyout": {
+                    "translations": [],
+                    "versions": [
+                        {"slug": "latest", "url": "/en/latest/"}
```

Comment:
We probably also want the name/verbose name here too, as we'll use that instead of slug for display purposes in the flyout menu.

Reply:
Yes. We will be using the `VersionSerializer` here for all of these versions.